src/systems/systems/src/air_conditioning/mod.rs

```diff
@@ -237,6 +315,396 @@ impl From<OverheadFlowSelector> for Ratio {
+                if total_air_mass_flow == MassRate::new::<kilogram_per_second>(0.) {
+                    ThermodynamicTemperature::new::<degree_celsius>(24.) // TODO: What should happen here?
+                } else {
+                    ThermodynamicTemperature::new::<kelvin>(
+                        total_mass_times_temp / total_air_mass_flow.get::<kilogram_per_second>(),
+                    )
+                };
+
+            self.outlet_air.set_flow_rate(total_air_mass_flow);
+            self.outlet_air.set_temperature(exit_temperature);
+
+            let outlet: Air = self.outlet_air;
+            self.individual_outlets
+                .iter_mut()
+                .for_each(|o| o.update(outlet));
+        }
+    }
+
+    fn mixer_unit_individual_outlet(&self, zone_id: usize) -> MixerUnitOutlet<ZONES> {
+        self.individual_outlets[zone_id]
+    }
+
+    fn outlet_temperature(&self) -> ThermodynamicTemperature {
+        self.outlet_air.temperature()
+    }
+}
+
+impl<const ZONES: usize> OutletAir for MixerUnit<ZONES> {
+    fn outlet_air(&self) -> Air {
+        self.outlet_air
+    }
+}
+
+#[derive(Clone, Copy)]
+struct MixerUnitOutlet<const ZONES: usize> {
+    outlet_air: Air,
+}
+
+impl<const ZONES: usize> MixerUnitOutlet<ZONES> {
+    fn new() -> Self {
+        Self {
+            outlet_air: Air::new(),
+        }
+    }
+
+    fn update(&mut self, mixer_unit_outlet: Air) {
+        self.outlet_air = mixer_unit_outlet;
+        self.outlet_air
+            .set_flow_rate(mixer_unit_outlet.flow_rate() / ZONES as f64)
+    }
+}
+
+impl<const ZONES: usize> OutletAir for MixerUnitOutlet<ZONES> {
+    fn outlet_air(&self) -> Air {
+        self.outlet_air
+    }
+}
+
+/// Temporary struct until packs are fully simulated
+struct AirConditioningPack {
+    outlet_air: Air,
+}
+
+impl AirConditioningPack {
+    fn new() -> Self {
+        Self {
+            outlet_air: Air::new(),
+        }
+    }
+
+    /// Takes the minimum duct demand temperature as the pack outlet temperature. This is accurate to real world behaviour but
+    /// this is a placeholder until the packs are modelled
+    fn update(&mut self, pack_flow: MassRate, duct_demand: &[ThermodynamicTemperature]) {
+        self.outlet_air.set_flow_rate(pack_flow);
+
+        let min_temp = duct_demand
+            .iter()
+            .fold(f64::INFINITY, |acc, &t| acc.min(t.get::<kelvin>()));
+        self.outlet_air
+            .set_temperature(ThermodynamicTemperature::new::<kelvin>(min_temp));
+    }
+}
+
+impl OutletAir for AirConditioningPack {
+    fn outlet_air(&self) -> Air {
+        self.outlet_air
+    }
+}
+
+struct TrimAirSystem<const ZONES: usize> {
+    duct_temperature_id: [VariableIdentifier; ZONES],
+    trim_air_valves: [TrimAirValve; ZONES],
+    // These are not a real components of the system, but a tool to simulate the mixing of air
+    pack_mixer_container: PneumaticPipe,
+    trim_air_mixer: [MixerUnit<1>; ZONES],
```

Comment:
I suggest using plural here since this is an array.

```suggestion
    trim_air_mixers: [MixerUnit<1>; ZONES],
```